tests/jeos/rpi_wifi.pm

```diff
@@ -40,6 +40,8 @@ sub run {
     enter_cmd(get_required_var('RPI_WIFI_PSK'));
 
     assert_screen 'text-logged-in-root';
+    script_run 'clear';
```

Comment:
That should be better `enter_cmd`

Reply:
why not check if it returned?

Reply:
You are ignoring the result code anyway, you wouldn't loose anything

Reply:
I ignore the result code but not the fact that it returned.
This prevents a race condition as it really waits until then before the next command is typed.